gensrc/script/doris_builtins_functions.py

```diff
@@ -176,6 +176,82 @@
         '_ZN5doris10vectorized18FunctionArrayIndexINS0_19ArrayPositionActionENS0_17NameArrayPositionEE12execute_implEPN9doris_udf15FunctionContextERNS0_5BlockERKSt6vectorImSaImEEmm',
         '', '', 'vec', ''],
 
+    [['array_min'], 'TINYINT', ['ARRAY_TINYINT'],
+        '_ZN5doris10vectorized19FunctionArrayMappedINS0_18ArrayAggregateImplILNS0_18AggregateOperationE0EEENS0_12NameArrayMinEE12execute_implEPN9doris_udf15FunctionContextERNS0_5BlockERKSt6vectorImSaImEEmm',
```

Comment:
the array function only support in `vec` query engine. maybe no need to add symbol?

Reply:
Yes, they are functions for `vec` query engine. Symbols are removed.